### PR TITLE
Replace DK-2 with ZH-1

### PR DESCRIPTION
**Pull request by ghost**
As we wind down the DK-2 datacenter and shift capacity to ZH-1, replace the datacenter description pages accordingly.

Once the hosts are fully up and running in ZH-1 and GVA-2, we'll replace the ping endpoints as well.

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -47,18 +48,14 @@
...
                     :optimizations   :advanced
                     :closure-defines {goog.DEBUG false}
-                    :pretty-print    false}}
+                    :pretty-print
 
-
-    ]}
-
-  )
+                    false}}]})
```
Comment (greut) [inline project.clj:61]: cljfmt, _shrugs_?

**Review by greut — CHANGES_REQUESTED** [review]
it's ZRH-1, DK-2 was the only two-letters data center.

File: project.clj (hunk 1 above)
Comment (ghost) [reply project.clj:61]: The original formatting was generated from the `lein new` invocation.  It should have been cleaned up long ago!

**Comment by ghost** [comment]
Thanks for the review @greut!

**Pull request merged**